packages/terra-table/docs/TableSectionHeader.md

```diff
@@ -0,0 +1,30 @@
+# Terra Table SectionHeader
+
+The Terra Section Header is built upon a table row and can be placed similarly within a table, provided the number of columns are provided to the SectionHeader. The section header should only be consumed directly when the header is static, as the actual row collapse is handled by the section.
```

Comment:
can be placed similarly within a table, p -> Might make sense to be placed within the table via children.

> The section header should only be consumed directly when the header is static, as the actual row collapse is handled by the section.

I see what you are saying here. It might be helpful to provide an example differentiating the two

Reply:
Updated Example.